src/v3/src/components/InformationalText/InformationalText.tsx

```diff
@@ -30,8 +30,6 @@ const InformationalText: UISchemaElementComponent<{
       <Typography
         paragraph
         data-se="o-form-explain"
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...(variant && { variant })}
```

Comment:
Variant is useless here because this component is used as a paragraph. It was being used for H* elements but we already have a dedicated component for that `Heading`. But ultimately did not need this.